src/events/command-handler/CorePreCommandRun.ts

```diff
@@ -25,7 +25,7 @@ export class CoreEvent extends Event<Events.PreCommandRun> {
 			return;
 		}
 
-		const result = await command.preconditions.run(message, command);
+		const result = await command.preconditions.run(message, command, { command });
```

Comment:
This context feels awfully redundant?

Reply:
It's more a "Run the preconditions for this command (second arg), and I am this source (third arg)", useful for running preconditions in the help command.